CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+This is a guide to help those who are interested in contributing to VitePress!
+
+## Prerequisites
+
+- [yarn](https://classic.yarnpkg.com/en/docs/cli/install/)
+
+## Instructions
+
+### Setup VitePress dev environment
+
+1. Clone the VitePress repo
+1. Install dependencies
+    ```
+    yarn
+    ```
+1. Create symlink to allow projects to link to local VitePress dev environment
+    ```bash
+    yarn link
+    ```
+    - If it's successful, you should see the following message:
+    ```
+    success Registered "vitepress".
+    info You can now run `yarn link "vitepress"` in the projects where you want to use this package and it will be used instead.
+    ✨  Done in 0.05s.
+    ```
+1. Start VitePress local dev environment
+    ```bash
+    yarn dev
+    ```
+
+
+
+### Setup local VitePress project
+
+1. Open up terminal
+1. Create a new folder
+1. Initialize with `npm init`
+1. Create a `docs` directory
+1. Create an `index.md` file with some content inside of `/docs`
+1. Add dependency to local VitePress dev environment
+    ```bash
+    yarn link vitepress
+    ```
+1. Add script to run VitePress in `package.json`
```

Comment:
I think it worth mentioning that before we do `vitepress dev`, we should be running `yarn dev` or `yarn build` at VitePress side to make it work. Tiny _note_ may be good?

**NOTE:** Remember to run `yarn dev` at VitePress project mentioned at "Setup VitePress dev environment" section before running `vitepress dev` at local project.